ArduPlane/quadplane.cpp

```diff
@@ -43,32 +43,32 @@ const AP_Param::GroupInfo QuadPlane::var_info[] = {
     // @Path: ../libraries/AC_AttitudeControl/AC_PosControl.cpp
     AP_SUBGROUPPTR(pos_control, "P", 17, QuadPlane, AC_PosControl),
 
-    // @Param: VELZ_MAX
+    // @Param: PILOT_SPD_UP
```

Comment:
I like the name change, but being in cm/s matches `Q_WP_SPEED_UP`. It might be better to delay this change until we can change the WP units as well so they stay in sync.

Reply:
Same for `PILOT_SPD_DN` and `PILOT_ACCEL_Z`

Reply:
we already have mixed m/s and cm/s for quadplanes, I think this is an incremental improvement. I think if we delay this one then the same reasoning would lead us to delay many others too as flow on from this one.
The rename should make it clear the user needs to look at the param docs/units